libnmstate/nm/bond.py

```diff
@@ -97,6 +97,15 @@ def _get_options(nm_device):
     # Workaround of https://bugzilla.redhat.com/show_bug.cgi?id=1806549
     if "miimon" not in options:
         options["miimon"] = bond_setting.get_option_default("miimon")
+    # When arp_interval=0(disabled), NetworkManager does not save
+    # arp_ip_target="" in on-disk profile which will cause nmstate verification
+    # failure. Including "arp_ip_target" if "arp_interval" is found fix this
+    # problem.
```

Comment:
Is this a known bug in NM? If so, this is a workaround AFAICS, so please include the bugzilla ticket.

Reply:
I don't think it's a bug of NM. It's reasonable to say `arp_ip_target=""` when arp_interval is disabled.

Reply:
On the second through, yes, you made a point.

we fix it in `libnmstate/nm/` folder if that is a NM bug. If not, it should be in `state.py` as a common bond option.